br/pkg/lightning/checkpoints/checkpoints_sql_test.go

```diff
@@ -431,23 +431,23 @@ func TestDestroyAllErrorCheckpoints_SQL(t *testing.T) {
 
 	s.mock.ExpectBegin()
 	s.mock.
-		ExpectQuery("SELECT (?s:.+)'all' = \\?").
-		WithArgs(sqlmock.AnyArg()).
+		ExpectQuery("\\QSELECT t.table_name, COALESCE(MIN(e.engine_id), 0), COALESCE(MAX(e.engine_id), -1) FROM `mock-schema`.table_v8 t LEFT JOIN `mock-schema`.engine_v5 e ON t.table_name = e.table_name WHERE ? = ? AND t.status <= ? GROUP BY t.table_name\\E").
+		WithArgs(sqlmock.AnyArg(), sqlmock.AnyArg(), sqlmock.AnyArg()).
 		WillReturnRows(
 			sqlmock.NewRows([]string{"table_name", "__min__", "__max__"}).
 				AddRow("`db1`.`t2`", -1, 0),
 		)
 	s.mock.
-		ExpectExec("DELETE FROM `mock-schema`\\.chunk_v\\d+ WHERE table_name IN .+ 'all' = \\?").
```

Comment:
for these test cases, keep the SQL pattern. Don't checking too specificly because in future we may change chunk_v5 to chunk_v6

Reply:
modified